beluga_example/README.md

```diff
@@ -6,17 +6,22 @@ This package contains example launch files that demonstrate the use of Beluga-ba
 
 See the [getting started](../GETTING_STARTED.md) tutorial to setup a development container or install the package and dependencies from source.
 
-- Launch a pre-recorded ROS bag.
+- Launch a pre-recorded ROS bag with perfect odometry and Beluga AMCL.
   ```bash
-  ros2 launch beluga_example example_rosbag_launch.py
+  ros2 launch beluga_example perfect_odometry.launch.xml
   ```
 
-- Launch a simulation that can be teleoperated.
+- Launch a pre-recorded ROS bag and Beluga AMCL as a composable node.
   ```bash
-  ros2 launch beluga_example example_launch.py
+  ros2 launch beluga_example perfect_odometry.launch.xml use_composition:=True
   ```
 
-- Launch a simulation with Beluga AMCL as a composable node.
+- Launch a simulation that can be teleoperated together with Beluga AMCL.
   ```bash
-  ros2 launch beluga_example example_component_launch.py
+  ros2 launch beluga_example simulation.launch.xml
+  ```
+
+- Launch Beluga AMCL, a map server and a lifecycle manager, useful for testing on real robots.
```

Comment:
@nahueespinosa nit^2: 
```suggestion
- Launch Beluga AMCL, a map server, and a lifecycle manager. Useful for testing on real robots.
```

Reply:
Done in https://github.com/Ekumen-OS/beluga/pull/211/commits/2dcf64926eab12eaa968be560ba444e00700692b.